.travis.yml

```diff
@@ -1,17 +1,17 @@
 language: ruby
 sudo: false
-rvm: 2.4.1
+rvm: 2.4.2
 
 matrix:
   include:
-    - rvm: 2.3.4
-      env: "BLACKLIGHT_VERSION=5.17.2"
+    - env: "RAILS_VERION=5.1.4"
+    - rvm: 2.3.5
+      env: "BLACKLIGHT_VERSION=5.17.2 RAILS_VERSION=5.0.6"
```

Comment:
Is this mismatch of RAILS_VERSION intentional?

Reply:
Yes, I believe so.  This change is just a backport of an existing commit on master (which I didn't write).  I was thinking of making a new PR that updates the travis.yml on master which could be backported too if necessary.

Reply:
Sorry not sure if mismatch was spelling or version number but just in case, is VERION correct or should be VERSION?

Reply:
Nice catch. Many eyes make... light code review?

Reply:
Great catch!